content/appleboy/drone-scp/index.md

```diff
@@ -0,0 +1,197 @@
+---
+date: 2017-01-06T00:00:00+00:00
+title: SCP
+author: appleboy
+tags: [ ssh, scp ]
+repo: appleboy/drone-scp
+logo: scp.svg
+image: appleboy/drone-scp
+---
+
+The SCP plugin copy files and artifacts to target host machine via SSH. The below pipeline configuration demonstrates simple usage:
+
+```yaml
+pipeline:
+  scp:
+    image: appleboy/drone-scp
+    host: example.com
+    target: /home/deploy/web
+    source: release.tar.gz
+```
+
+Example configuration with custom username, password and port:
+
+```diff
+pipeline:
+  scp:
+    image: appleboy/drone-scp
+    host: example.com
++   username: appleboy
++   password: 12345678
++   port: 4430
+    target: /home/deploy/web
+    source: release.tar.gz
+```
+
+Example configuration with multiple source and target folder:
+
+```diff
+pipeline:
+  scp:
+    image: appleboy/drone-scp
+    host: example.com
+    target: 
++     - /home/deploy/web1
++     - /home/deploy/web2
+    source: 
++     - release_1.tar.gz
++     - release_2.tar.gz
+```
+
+Example configuration with multiple host:
+
+```diff
+pipeline:
+  scp:
+    image: appleboy/drone-scp
+-   host: example.com
++   host:
++     - example1.com
++     - example2.com
+    target: /home/deploy/web
+    source: release.tar.gz
+```
+
+Remove target folder before copy files and artifacts to target:
+
+```diff
+  scp:
+    image: appleboy/drone-scp
+    host: example.com
+    target: /home/deploy/web
+    source: release.tar.gz
++   rm: true
+```
+
+Example configuration for success build:
+
+```diff
+pipeline:
+  scp:
+    image: appleboy/drone-scp
+    host: example.com
+    target: /home/deploy/web
+    source: release.tar.gz
++   when:
++     status: success
+```
+
+Example configuration for tag event:
+
+```diff
+pipeline:
+  scp:
+    image: appleboy/drone-scp
+    host: example.com
+    target: /home/deploy/web
+    source: release.tar.gz
++   when:
++     status: success
++     event: tag
+```
+
+# Secrets
+
+The SCP plugin supports reading credentials from the Drone secret store. This is strongly recommended instead of storing credentials in the pipeline configuration in plain text.
+
+```diff
+pipeline:
+  scp:
+    image: appleboy/drone-scp
+    host: example.com
+    username: appleboy
+-   password: 12345678
+    port: 4430
+    target: /home/deploy/web
+    source: release.tar.gz
+```
+
+The above webhook Yaml attribute can be replaced with the `SCP_PASSWORD` secret environment variable. Please see the Drone documentation to learn more about secrets.
```

Comment:
we are making some changes to secrets in drone 0.6 and therefore have decided not to put example CLI commands in the documentation. Can you model this section after this example? https://github.com/drone/drone-plugin-index/blob/master/content/drone-plugins/drone-s3/index.md#secrets